Makefile

```diff
@@ -120,6 +120,7 @@ install.runc:
 
 .PHONY: test-integration
 test-integration:
+	#ginkgo -v -cover -progress -trace tests/e2e/.
```

Comment:
Why is this commented out?  It might help others understand, if the reason was also commented :smile:

Reply:
I commented it out as I was apparently having trouble creating the output file that gets shipped out to codecov.io  and I was trying to limit the number of places it might get created.  One of those "pushing this button didn't work, let's try pushing that one!" type of debug sessions.  Long-term I'd have that turned on if possible.